rhscl-postgresql-apb/roles/rhscl-postgresql-apb-openshift/tasks/main.yml

```diff
@@ -0,0 +1,97 @@
+- name: create namespace
+  openshift_v1_project:
+    name: '{{ namespace }}'
+
+- name: create volumes
+  k8s_v1_persistent_volume_claim:
+    name: postgresql
+    namespace: '{{ namespace }}'
+    state: present
+    access_modes:
+      - ReadWriteOnce
+    resources_requests:
+      storage: 1Gi
+
+- name: create service
+  k8s_v1_service:
+    name: postgresql
+    namespace: '{{ namespace }}'
+    state: present
+    labels:
+      app: rhscl-postgresql-apb
+      service: postgresql
+    selector:
+      app: rhscl-postgresql-apb
+      service: postgresql
+    ports:
+    - name: port-5432
+      port: 5432
+      protocol: TCP
+      target_port: 5432
+  register: postgres_service
+
+- name: create deployment config
+  openshift_v1_deployment_config:
+    name: postgresql
+    namespace: '{{ namespace }}'
+    labels:
+      app: rhscl-postgresql-apb
+      service: postgresql
+    replicas: 1
+    selector:
+      app: rhscl-postgresql-apb
+      service: postgresql
+    strategy_type: Rolling
+    strategy_rolling_params:
+      interval_seconds: 1
+      max_surge: 25%
+      max_unavailable: 25%
+      timeout_seconds: 600
+      update_period_seconds: 1
+    spec_template_metadata_labels:
+      app: rhscl-postgresql-apb
+      service: postgresql
+    containers:
+    - env:
+      - name: POSTGRESQL_PASSWORD
+        value: '{{ postgresql_password }}'
+      - name: POSTGRESQL_USER
+        value: '{{ postgresql_user }}'
+      - name: POSTGRESQL_DATABASE
+        value: '{{ postgresql_database }}'
+      image: registry.access.redhat.com/rhscl/{{ image }}
+      image_pull_policy: IfNotPresent
+      name: postgresql
+      ports:
+      - container_port: 5432
+        protocol: TCP
+      resources: {}
```

Comment:
I have no idea. They were included in the postgresql-apb example I took this from. I can try without them.